workflow-ui/backstack-android/api/backstack-android.api

```diff
@@ -31,36 +29,11 @@ public final class com/squareup/workflow1/ui/backstack/BackStackContainer$Compan
 	public fun getType ()Lkotlin/reflect/KClass;
 }
 
-public final class com/squareup/workflow1/ui/backstack/ViewStateCache : android/os/Parcelable {
-	public static final field CREATOR Lcom/squareup/workflow1/ui/backstack/ViewStateCache$CREATOR;
+public final class com/squareup/workflow1/ui/backstack/ViewStateCache {
 	public fun <init> ()V
-	public fun describeContents ()I
+	public final fun installOnContainer (Landroid/view/View;)V
```

Comment:
Really happy that I managed to whittle this public API down to all of 3 methods.